hass_nabucasa/voice.py

```diff
@@ -35,50 +35,137 @@ class Gender(str, Enum):
     ("zh-HK", Gender.FEMALE): "HiuMaanNeural",
     ("zh-HK", Gender.MALE): "WanLungNeural",
+    ("zh-CN", Gender.FEMALE): "XiaohanNeural",
+    ("zh-CN", Gender.FEMALE): "XiaomoNeural",
+    ("zh-CN", Gender.FEMALE): "XiaoruiNeural",
     ("zh-CN", Gender.FEMALE): "XiaoxiaoNeural",
+    ("zh-CN", Gender.FEMALE): "XiaoxuanNeural",
+    ("zh-CN", Gender.FEMALE): "XiaoyouNeural",
+    ("zh-CN", Gender.MALE): "YunxiNeural",
     ("zh-CN", Gender.MALE): "YunyangNeural",
+    ("zh-CN", Gender.MALE): "YunyeNeural",
     ("zh-TW", Gender.FEMALE): "HsiaoChenNeural",
+    ("zh-TW", Gender.FEMALE): "HsiaoYuNeural",
     ("zh-TW", Gender.MALE): "YunJheNeural",
     ("hr-HR", Gender.FEMALE): "GabrijelaNeural",
     ("hr-HR", Gender.MALE): "SreckoNeural",
     ("cs-CZ", Gender.FEMALE): "VlastaNeural",
     ("cs-CZ", Gender.MALE): "AntoninNeural",
     ("da-DK", Gender.FEMALE): "ChristelNeural",
     ("da-DK", Gender.MALE): "JeppeNeural",
+    ("nl-BE", Gender.FEMALE): "DenaNeural",
+    ("nl-BE", Gender.MALE): "ArnaudNeural",
     ("nl-NL", Gender.FEMALE): "ColetteNeural",
+    ("nl-NL", Gender.FEMALE): "FennaNeural",
     ("nl-NL", Gender.MALE): "MaartenNeural",
     ("en-AU", Gender.FEMALE): "NatashaNeural",
     ("en-AU", Gender.MALE): "WilliamNeural",
     ("en-CA", Gender.FEMALE): "ClaraNeural",
     ("en-CA", Gender.MALE): "LiamNeural",
+    ("en-HK", Gender.FEMALE): "YanNeural",
+    ("en-HK", Gender.MALE): "SamNeural",
     ("en-IN", Gender.FEMALE): "NeerjaNeural",
     ("en-IN", Gender.MALE): "PrabhatNeural",
     ("en-IE", Gender.FEMALE): "EmilyNeural",
     ("en-IE", Gender.MALE): "ConnorNeural",
-    ("en-GB", Gender.FEMALE): "MiaNeural",
+    ("en-KE", Gender.FEMALE): "AsiliaNeural",
+    ("en-KE", Gender.MALE): "ChilembaNeural",
+    ("en-NZ", Gender.FEMALE): "MollyNeural",
+    ("en-NZ", Gender.MALE): "MitchellNeural",
+    ("en-NG", Gender.FEMALE): "EzinneNeural",
+    ("en-NG", Gender.MALE): "AbeoNeural",
+    ("en-PH", Gender.FEMALE): "RosaNeural",
+    ("en-PH", Gender.MALE): "JamesNeural",
+    ("en-SG", Gender.FEMALE): "LunaNeural",
+    ("en-SG", Gender.MALE): "WayneNeural",
+    ("en-ZA", Gender.FEMALE): "LeahNeural",
+    ("en-ZA", Gender.MALE): "LukeNeural",
+    ("en-TZ", Gender.FEMALE): "ImaniNeural",
+    ("en-TZ", Gender.MALE): "ElimuNeural",
+    ("en-GB", Gender.FEMALE): "LibbyNeural",
+    # Retired on 30 October 2021, requests will be redirected to SoniaNeural
+    ("en-GB", Gender.FEMALE): "MiaNeural", 
+    ("en-GB", Gender.FEMALE): "SoniaNeural",
     ("en-GB", Gender.MALE): "RyanNeural",
+    ("en-US", Gender.FEMALE): "AmberNeural",
+    ("en-US", Gender.FEMALE): "AriaNeural",
+    ("en-US", Gender.FEMALE): "AshleyNeural",
+    ("en-US", Gender.FEMALE): "CoraNeural",
+    ("en-US", Gender.FEMALE): "ElizabethNeural",
     ("en-US", Gender.FEMALE): "JennyNeural",
+    ("en-US", Gender.FEMALE): "MichelleNeural",
+    ("en-US", Gender.FEMALE): "MonicaNeural",
+    ("en-US", Gender.FEMALE): "SaraNeural",
```

Comment:
Marking as a draft for now, since it seems that Nabu Casa service can only handle a single male / female voice. If this is correct, I will remove all duplicate voices. (also for other locales)